update/interface.go

```diff
@@ -227,3 +227,9 @@ type EpochChangeConfirmedNotifier interface {
 	RegisterForEpochChangeConfirmed(handler func(epoch uint32))
 	IsInterfaceNil() bool
 }
+
+// Closer defines the functionality of an entity that can be closed
```

Comment:
I already defined one in core/close. Maybe delete one of them.

Reply:
will link to that one. Having only primitive types in its description makes this interface "redeclarable" in all other packages.